pages/LoginView.neu.page.ts

```diff
@@ -0,0 +1,42 @@
+import { expect, Page } from '@playwright/test';
+import generator from 'generate-password-ts';
+import { StartViewPage } from './StartView.neu.page';
+
+export class LoginViewPage {
+  /* add global locators here */
+  readonly page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  /* actions */
+  public async waitForPageLoad(): Promise<void> {
+    await this.page.getByTestId('login-page-title').waitFor({ state: 'visible' });
+    await expect(this.page.getByTestId('login-page-title')).toHaveText('Anmeldung');
+  }
+
+  async login(
+    username: string = process.env.USER as string,
+    password: string = process.env.PW as string
+  ): Promise<StartViewPage> {
+    await expect(this.page.getByTestId('login-page-title')).toHaveText('Anmeldung');
+    await expect(this.page.getByTestId('login-prompt-text')).toHaveText('Bitte geben Sie Ihre persönlichen Zugangsdaten ein.');
+    await this.page.getByTestId('username-input').fill(username);
+    await this.page.getByTestId('password-input').fill(password);
+    await this.page.getByTestId('login-button').click();
```

Comment:
```suggestion
  
  const usernameInput = this.page.getByTestId('username-input');
  const passwordInput = this.page.getByTestId('password-input');
  const loginButton = this.page.getByTestId('login-button');

  await usernameInput.waitFor({ state: 'visible' });
  await usernameInput.fill(username);

  await passwordInput.waitFor({ state: 'visible' });
  await passwordInput.fill(password);

  await loginButton.click();
```
Maybe safer and cleaner this way? Also makes maintenance easier if test IDs change.